File: testdata/jennies/builders/properties/PHPBuilder/src/Properties/SomeStructBuilder.php
<?php

namespace Grafana\Foundation\Properties;

/**
 * @implements \Grafana\Foundation\Cog\Builder<\Grafana\Foundation\Properties\SomeStruct>
 */
class SomeStructBuilder implements \Grafana\Foundation\Cog\Builder
{
    protected \Grafana\Foundation\Properties\SomeStruct $internal;
    private string $someBuilderProperty;

    public function __construct()
    {
    	$this->internal = new \Grafana\Foundation\Properties\SomeStruct();
        $this->someBuilderProperty = "";
    }

    /**
     * Builds the object.
     * @return \Grafana\Foundation\Properties\SomeStruct
     */
    public function build()
    {
        return $this->internal;
    }

    public function id(int $id): static
    {
        $this->internal->id = $id;
    
        return $this;
    }

}

File: testdata/jennies/builders/properties/JavaBuilder/properties/SomeStructBuilder.java
package properties;


public class SomeStructBuilder implements cog.Builder<SomeStruct> {
    protected final SomeStruct internal;
    private String someBuilderProperty;
    
    public SomeStructBuilder() {
        this.internal = new SomeStruct();
    this.someBuilderProperty = "";
    }
    public SomeStructBuilder id(Long id) {
        this.internal.id = id;
        return this;
    }
    public SomeStruct build() {
        return this.internal;
    }
}
